Module 10 - Javascript and Ajax/classes/DeleteStudentController.cls

```diff
@@ -0,0 +1,29 @@
+public class DeleteStudentController
+{ 
+    ApexPages.StandardSetController setCon; 
+    public Boolean isNotEmpty {get; set;} 
+    public Boolean isEmpty {get; set;} 
```

Comment:
do we need two different variable for `isEmpty` and  `isNotEmpty` aren't they complimentary to each other